sdk/samples/iot/docs/how_to_iot_hub_samples_windows.md

```diff
@@ -1,449 +1,254 @@
 # How to Setup and Run Azure SDK for Embedded C IoT Hub Samples on Microsoft Windows
 
-This is a step-by-step documentation of how to start from scratch and get the Azure SDK for Embedded C IoT Hub Samples running on Microsoft Windows.
+## Introduction
 
-Prerequisites:
+This is a step-by-step guide of how to start from scratch and get the Azure SDK for Embedded C IoT Hub Certificate Samples running on Microsoft Windows.
 
-- [Having created an Azure account](https://azure.microsoft.com/en-us/)
-- [Having created an Azure IoT Hub](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal)
-- Microsoft Visual Studio installed in the local machine.
+Samples are designed to highlight the function calls required to connect with the Azure IoT Hub. These calls illustrate the happy path of the [mqtt state machine](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/iot/mqtt_state_machine.md). As a result, **these samples are NOT designed to be used as production-level code**. Production code needs to incorporate other elements, such as connection retries and more extensive error-handling, which these samples do not include. These samples also utilize OpenSSL, which is **NOT recommended to use in production-level code on Windows or macOS**.
 
-What is covered:
+For Windows, the command line examples are based on PowerShell. While Windows  devices are not likely to be considered constrained, these samples enable one to test the Azure SDK for Embedded C libraries, even without a real device.
 
-- Downloading and building the Azure SDK for Embedded C suite
-- Configuring and running the IoT Hub client samples.
+### WARNING: Samples are generic and should not be used in any production-level code.
```

Comment:
The heading level for such a warning is a bit high, imo. It shouldn't be at the same level as actual content like `Prerequisites`
```suggestion
***WARNING: Samples are generic and should not be used in any production-level code.***
```

Also, I don't think we should use headings for things like this (which creates an anchor link).

Reply:
agreed, and in fact i just changed that a few minutes ago! good eye.